runtime-sdk/src/modules/core/mod.rs

```diff
@@ -268,6 +331,11 @@ impl module::MethodRegistrationHandler for Module {
             name: "core.EstimateGas",
             handler: Self::_query_estimate_gas_handler,
         });
+
+        methods.register_query(module::QueryMethodInfo {
+            name: "core.BatchWeightLimits",
```

Comment:
May be useful to define this method name in `oasis-core-runtime` as it will be part of the Host API?

Reply:
yeah agreed